lib/ui/natives.dart

```diff
@@ -36,6 +36,10 @@ void _setupHooks() {
   }());
 }
 
+/// Saves Dart compilation trace to a file.
+///
+/// The output is a list of symbols compiled by the Dart JIT at runtime up to this point,
+/// and can be passed later to Dart tools in order to precompile this code offline.
 void saveCompilationTrace(String filePath) {
```

Comment:
do we need this if we have `dumpCompilationTrace`?

Reply:
Renamed dumpCompilationTrace to saveCompilationTrace. The original saveCompilationTrace was only used by an old internal sample app, and it's OK to break it.